core/src/test/java/com/whylogs/core/DatasetProfileTest.java

```diff
@@ -1,15 +1,5 @@
 package com.whylogs.core;
 
-import com.google.common.collect.ImmutableMap;
```

Comment:
IDE formatting settings? We probably want to check these in/align these

Reply:
I did not change these because if IDE settings.
The spotless check insisted I do so,
Did you change the spotsless checks after DatasetProfileTest.java had already been checked in?

https://github.com/whylabs/whylogs-java/pull/21/checks?sha=2af91680300a0a7ec32d2e50e2c1e3a878576fd0

Reply:
Okay that's weird that it got checked in and bypassed the Git workflow. Maybe the flow doesn't trigger splotlessApply